docs/sphinx/api/public_api.rst

```diff
@@ -529,8 +529,8 @@ more information about the parallel algorithms.
 Functions
 ---------
 
-- :cpp:func:`hpx::parallel::v1::uninitialized_copy`
-- :cpp:func:`hpx::parallel::v1::uninitialized_copy_n`
+- :cpp:func:`hpx::uninitialized_copy`
+- :cpp:func:`hpx::uninitialized_copy_n`
```

Comment:
Could you please add the `hpx::ranges` overloads as well, similarly to how we have them listed in `hpx/algorithm.hpp`?